src/main/java/uk/ac/ebi/embl/api/validation/submission/ServiceConfig.java

```diff
@@ -0,0 +1,32 @@
+package uk.ac.ebi.embl.api.validation.submission;
+
+public class ServiceConfig {
```

Comment:
If you are including spring libraries you could read this information from a spring property file